### PR TITLE
Changed dogehouse logo in readme

### PR DESCRIPTION
**Pull request by vi241**

### CODE REVIEW

File: 🐕.🏠 [hunk 1]
```diff
@@ -1 +1 @@
-🐕🏠🚀🌕
+🐕🏠🚀🌕
```
Comment (overlisted) [inline 🐕.🏠:1]: huh

Comment (vi241) [reply 🐕.🏠:1]: idk

Comment (jamesql) [reply 🐕.🏠:1]: big change right here, must review further

Comment (vi241) [reply 🐕.🏠:1]: lol

File: README.md [hunk 2]
```diff
@@ -65,6 +65,9 @@ Contributors helping to review/merge pull requests:
 - @ofsho
 - @HarrisonMayotte
 - @nadirabbas
+- @overlisted
+- @goldyydev
+- @jamesql
```
Comment (amitojsingh366) [inline README.md:70]: forgot to add me

File: 🐕.🏠 (hunk 1 above)
Comment (amitojsingh366) [reply 🐕.🏠:1]: pretty sure it was `🐕🏠-🚀🌕` not `🐕🏠🚀🌕`

File: README.md (hunk 2 above)
Comment (vi241) [reply README.md:70]: i thought you were there

File: 🐕.🏠 (hunk 1 above)
Comment (vi241) [reply 🐕.🏠:1]: it was i changed

**Pull request merged**